### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by samchrisinger**
# Purpose

Adds components for the various OSF footers.

### CODE REVIEW

File: addon/components/osf-mode-footer/style.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#devmode {
+    position:fixed;
+    bottom:0;
+    left:0;
+    border-top-right-radius:8px;
+    background-color:red;
+    color:white;
+    padding:.5em;
+}
+
```
Comment (abought) [inline addon/components/osf-mode-footer/style.scss]: Linebreak at end of file please

File: addon/components/osf-mode-footer/style.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#devmode {
+    position:fixed;
+    bottom:0;
+    left:0;
+    border-top-right-radius:8px;
+    background-color:red;
+    color:white;
+    padding:.5em;
+}
```
Comment (abought) [inline addon/components/osf-mode-footer/style.scss]: Linebreak here please

File: addon/components/osf-footer/component.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import config from 'ember-get-config';
+
+export default Ember.Component.extend({
+    layout,
+    helpUrl: config.OSF.helpUrl,
+    supportEmail: Ember.computed(function() {
+        return '<n uers=\"znvygb:pbagnpg@bfs.vb\" ery=\"absbyybj\">Pbagnpg</n>'.replace(
+            /[a-zA-Z]/g,
+            function(e) {
+                return String.fromCharCode((e <= 'Z' ? 90 : 122) >= (e = e.charCodeAt(0) + 13) ? e : e - 26);
```
Comment (abought) [inline addon/components/osf-footer/component.js]: Given that the entire page is *already* constructed from JS, I'm not sure what ROT13 gets us here in terms of protecting email addresses.

However, it's not a blocker for the PR; I know we're hewing close to the original.

**Comment by abought** [comment]
The copyright footer is a bit off-center on the login page. 

I only noticed it because it was covered by the DEVMODE label on a small screen.

screen shot 2016-06-23 at 4 37 23 pm

**Comment by abought** [comment]
Pass complete. Two basic notes. Neither blocking, but both nice to have. 

Will leave merge timing to developer discretion.

**Comment by samchrisinger** [comment]
I think offsetting issues will be fixed when we import OSF's style.css.

**Pull request merged**